russian/glue/wallpapers.md

```diff
@@ -0,0 +1,9 @@
+Да здесь всё так липко, и мне не соствит труда этого сделать.
+С чего начать? Ах да, мне же понадобятся некоторые инструменты да и обои.
+Какие бы мне выбрать?
+[эти](https://www.google.be/search?q=windows&tbm=isch) или может вот [эти](https://www.google.be/search?q=linux&tbm=isch)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.